src/PagerScroll.js

```diff
@@ -152,8 +153,8 @@ export default class PagerScroll<T: *> extends React.Component<
         overScrollMode="never"
         scrollEnabled={this.props.swipeEnabled}
         automaticallyAdjustContentInsets={false}
-        bounces={false}
-        alwaysBounceHorizontal={false}
+        bounces={bouncesEnabled || false} // default false
```

Comment:
We can use `defaultProps` to set it to `false` instead

Reply:
nice. It is better.